site/docs/.vuepress/config.ts

```diff
@@ -385,6 +385,10 @@ export default defineUserConfig({
                     text: "Virtual Private Server",
                     link: "/hosting/vps.html",
                   },
+                  {
```

Comment:
I'd prefer it if VPS would remain as the last entry. I don't care much about the order of the other platforms.

Reply:
Sure, no problem!

Reply:
@jgcarrillo Please click "Resolve conversation" on the conversations you have took action about.